src/praisonai-bot/praisonai_bot/bots/discord.py

```
@@ -676,6 +683,76 @@ async def health(self):
+            if not re.fullmatch(r"[a-z0-9_-]{1,32}", safe_name):
+                continue
+            desc = (str(description) or "").strip()[:100] or safe_name
+
+            async def _callback(interaction, _name=safe_name, _prefix=prefix):
+                try:
+                    await interaction.response.send_message(
+                        f"Type `{_prefix}{_name}` as a message to run this command.",
+                        ephemeral=True,
+                    )
+                except Exception:  # noqa: BLE001 — never break the interaction
+                    pass
```

Ruff (0.15.20)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)